types/index.d.ts

```diff
@@ -4,7 +4,7 @@ declare module 'found' {
   import * as React from 'react';
   import { Reducer, Store, StoreEnhancer } from 'redux';
 
-  type Omit<T, K extends keyof T> = Pick<T, Exclude<keyof T, K>>;
+  type Omit<T, K extends keyof any> = Pick<T, Exclude<keyof T, K>>;
```

Comment:
Omit is upstream in TS and it behaves like that

Reply:
it extends `keyof any` i think

Reply:
right. I mean, use the upstream.

Reply:
these ts defs are supposed to work with v3.0. so for 3.0 through 3.4, we can't